wsrep_api.h

```diff
@@ -63,7 +63,7 @@ extern "C" {
  *                                                                        *
  **************************************************************************/
 
-#define WSREP_INTERFACE_VERSION "26"
+#define WSREP_INTERFACE_VERSION "26.1"
```

Comment:
I'm not sure if we have to change the verion, codership also added methods without touching it.

Reply:
What is see is that struct wsrep_st is not changed without api version change.
What they do is adding server side callbacks without api version bumping, which is relatively safe, because if server doesn't register its callback, the library should not attempt to call them (at least in theory).
The problem starts when we change struct wsrep_st in the way when we add something in the middle (or change the method signature) without bumping api version. Server will access methods in the "old way", while underlying wsrep_st structure is not compatible.
We added several methods before to wsrep_st struct without bumping interface version. This is wrong in my opinion. the purpose of this string is to forbid loading incompatible library. Now we've got the situation when it is possible to load vanilla upstream library, but it will obviously crash because of ABI differences in `struct wsrep_st`